src/kv/store.h

```diff
@@ -1205,5 +1209,15 @@ namespace kv
       // is incremented.
       return ReservedTx(this, term_of_last_version, tx_id);
     }
+
+    virtual void set_flags(uint8_t flags) override
```

Comment:
This will probably need a lock as the `store` is accessed by multiple transactions, unless this is called from a location where the lock is already taken, in which case `_unsafe` should be appended to the function name.